### ISSUE TITLE
Typescript is unable to locate angulartics2 package

### ISSUE THREAD

**Issue by mahesh-kedari** [NathanWalker/angular-seed-advanced#288]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here, instead see use [gitter](https://gitter.im/mgechev/angular2-seed) or [stackoverflow](https://stackoverflow.com/questions/tagged/angular2)
```

**Current behavior**

Getting an error message on console while running **npm start**
src\client\app\frameworks\analytics\index.ts(3,37): error TS2307: Cannot find module 'angulartics2/src/providers/angulartics2-segment'.
src\client\app\frameworks\analytics\services\analytics.service.ts(9,37): error TS2307: Cannot find module 'angulartics2/src/providers/angulartics2-segment'.

**Expected behavior**

**Minimal reproduction of the problem with instructions**

**What is the motivation / use case for changing the behavior?**

**Please tell us about your environment:**

Windows 8.1
Visual Studio Code 1.7.1
Node 6.9.1
npm 3.7.2
Typescript version: 2.0.7

* **Angular Seed Version:** 2.0.X

* **Node:** `node --version` = 6.9.1

**Comment by mahesh-kedari** [comment]
Root cause is change in type definition file of Angulartics2. 

Required code changes.
src/client/app/frameworks/analytics/index.ts

```
import { Angulartics2, Angulartics2Segment } from 'angulartics2';
//import { Angulartics2Segment } from 'angulartics2/dist/providers/angulartics2-segment';

```

**Comment by gucolin** [comment]
The fix failed prod build. Error:

`
Error on fetch for angulartics2/dist/providers.js at file:///***/angular2-seed-advanced/node_modules/angulartics2/dist/providers.js
        Loading angulartics2
        Loading dist/tmp/app/frameworks/analytics/index.js
        Loading dist/tmp/app/components/app.component.js
        Loading dist/tmp/web.module.js
        Loading dist/tmp/main.web.js
        Error: ENOENT: no such file or directory, open '***/angular2-seed-advanced/node_modules/angulartics2/dist/providers.js'
       at Error (native)
`

**Comment by Kaffiend** [comment]
Ill look into why e2e is failing, i cant find anything at the moment that would cause it though, maybe its just a travis thing, im not sure. But the PR does fix the prod.build issue with the previous Merge. And you can override the default from the project config as well with a different provider name. The way the Angulartics2 package is structured makes it simple.

**Comment by Kaffiend** [comment]
#290 fixes this.

**Comment by melliott03** [comment]
This worked for me...
navigate to:          nativescript/app/shared/nativescript/index.ts

do the following edit:
// import { Angulartics2Segment } from 'angulartics2/src/providers/angulartics2-segment';
import { Angulartics2Segment } from 'angulartics2/dist/providers/segment/angulartics2-segment';

**Comment by Kaffiend** [comment]
#290 only fixed the web references. I didnt touch the nativescript so those still need updating. 

Thanks for catching that @melliott03

**Comment by NathanWalker** [comment]
Fixed on master now with angulartics2 `1.4.0` update.
